subscriptions/subscriptions-impl/src/main/java/com/duckduckgo/subscriptions/impl/SubscriptionsManager.kt

```diff
@@ -248,6 +347,10 @@ class RealSubscriptionsManager @Inject constructor(
             null
         }
     }
+
+    companion object {
+        const val SUBSCRIPTION_NOT_FOUND = "SubscriptionNotFound"
```

Comment:
nit: Suffix with `_ERROR` ? I guess the idea is that more of these are added?

```suggestion
        const val SUBSCRIPTION_NOT_FOUND_ERROR = "SubscriptionNotFound"
```